src/screens/Login/index.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from "react";
+import {
+  HelpBlock,
+  FormGroup,
+  FormControl,
+  ControlLabel
+} from "react-bootstrap";
+import { Auth } from 'aws-amplify';
+import LoaderButton from '../../components/LoaderButton';
+import "./index.css";
+
+class Login extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
```

Comment:
Can do this outside of the constructor declaration, and get rid of the `constructor` method completely

Reply:
I have moved the state initialize outside and removed the constructor.